dts/common/nordic/nrf54h20_eng0.dtsi

```diff
@@ -0,0 +1,309 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <mem.h>
+#include <nordic/nrf_common.dtsi>
+
+#include <zephyr/dt-bindings/misc/nordic-nrf-ficr-nrf54h20-eng0.h>
+
+/delete-node/ &sw_pwm;
+
+/ {
+	#address-cells = <1>;
+	#size-cells = <1>;
+
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		cpuapp: cpu@2 {
+			compatible = "arm,cortex-m33";
+			reg = <2>;
+			device_type = "cpu";
+			clock-frequency = <DT_FREQ_M(320)>;
+		};
+
+		cpurad: cpu@3 {
+			compatible = "arm,cortex-m33";
+			reg = <3>;
+			device_type = "cpu";
+			clock-frequency = <DT_FREQ_M(256)>;
+		};
+
+		cpuppr: cpu@d {
+			compatible = "nordic,riscv";
```

Comment:
Let's add a separate binding in `dts/bindings/cpu`  that inherits from the common `riscv,cpus.yaml`, so that the `riscv,isa` property is properly enforced

Reply:
Applied.